.github/kokoro/ql.sh

```diff
@@ -2,8 +2,9 @@
 
 SCRIPT_SRC="$(realpath ${BASH_SOURCE[0]})"
 SCRIPT_DIR="$(dirname "${SCRIPT_SRC}")"
+INSTALL_DIR="$(pwd)/github/${KOKORO_DIR}/install"
 
-export CMAKE_FLAGS="-GNinja"
+export CMAKE_FLAGS="-GNinja -DINSTALL_FAMILIES=qlf_k4n8 -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR}"
```

Comment:
INSTALL_FAMILIES flag usage seems to be inconsistent. In 7 series FAMILY is the whole xc7, while devices are selected with INSTALL_DEVICES. Here `INSTALL_FAMILIES` is used to select a single device

Reply:
It looks that way but in this case `qlf_k4n8` refers both to the device and the family (same name). See: https://github.com/antmicro/symbiflow-arch-defs/blob/4b4854ef256a4de4a3672e6b1f076161a11b56ef/quicklogic/qlf_k4n8/CMakeLists.txt#L6

The family may be renamed to eg. `qlf` but I suppose that it will come later as more QuickLogic architectures are added.